.github/workflows/pull_request.yaml

```diff
@@ -93,18 +79,15 @@ jobs:
         if: matrix.env.TEST_TYPE == 'release-upgrade' || matrix.env.TEST_TYPE == 'source-upgrade'
         run: |
           docker pull quay.io/pulp/pulp-ci-dbuster:3.0.0
-          docker pull quay.io/pulp/pulp_rpm-ci-f31:3.1.0
-          docker pull quay.io/pulp/pulp_rpm-ci-c7:3.1.0
+          docker pull quay.io/pulp/pulp-ci-c7:3.0.0
+          docker pull quay.io/pulp/pulp-ci-f31:3.0.0
```

Comment:
Why this change? We still need to test upgrading systems with pulp_rpm installed. That's what pulp_rpm-ci images do.

Note that I wrote their description on quay.io.